logfire/_internal/main.py

```diff
@@ -1451,12 +1452,31 @@ def instrument_sqlite3(
             },
         )
 
+    def instrument_aws_lambda(self, lambda_function: Any, **kwargs: Unpack[AwsLambdaInstrumentKwargs]) -> None:
```

Comment:
Can this be type-hinted?

Reply:
`Callable[[Any, Any], Any]`, or maybe `Callable[[dict[str, Any], dict[str, Any]], Any]`...

Reply:
I'm not 100% sure if the one on the left is correct tho...

Reply:
I think the basic one on the left is fine. Also maybe a note in docs about this param.

Reply:
What do you want to mention?

Reply:
I guess just mention what they're supposed to pass in